src/components/popup-menu/popup-menu-item.jsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import injectSheet from 'react-jss';
+import cn from 'classnames';
 import styles from './popup-menu.styles';
 
-const PopupMenuItem = ({ node, children, topBorder, linkTo, onClick, onKeyDown, listItemRef, disabled, classes, onFocus }) => {
+const PopupMenuItem = ({ node, children, topBorder, linkTo, onClick, onKeyDown, listItemRef, disabled, classes, onFocus, className }) => {
   const Element = disabled ? 'button' : node;
+  const classNameStr = className ? cn(classes.item, className) : classes.item;
   return (
-    <li className={classes.item}>
+    <li className={classNameStr}>
```

Comment:
You could remove line 9 and just write `<li className={cn(classes.item, className)}>` here